doc/src/modules/physics/mechanics/masses.rst

```diff
@@ -21,7 +21,7 @@ Particle
 
 Particles are created with the class ``Particle`` in :mod:`mechanics`.
 A ``Particle`` object has an associated point and an associated mass which are
-the only two attributes of the object.::
```

Comment:
Odd, I was under the impression that the double colon signified a code block in restructuredtext. Maybe Sphinx makes some different assumptions?

See here: http://www.sphinx-doc.org/en/stable/rest.html#source-code

Seems like they are required...